flight/Modules/Actuator/actuator.c

```diff
@@ -761,24 +872,141 @@ static float collective_curve(float const input, float const * curve, uint8_t nu
 	return linear_interpolate(input, curve, num_points, -1.0f, 1.0f);
 }
 
+static float scale_channel_dshot(float value, int idx, bool active_command)
```

Comment:
Now when we're sending dshot, we don't have such a reliance on the MAX being a proper value, and neutral being in a legal range.  We really just calculate the correct values directly.